spec/libraries/aws_engine_spec.rb

```diff
@@ -68,18 +73,15 @@ def initialize
           @contents.push(obj)
         end
 
-        def contents
-          @contents
```

Comment:
__[reek]__
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)